documentation/library-reference/source/system/file-system.rst

```diff
@@ -186,6 +188,20 @@ File-System module.
      refers to a link, the link is removed, but the actual file that the
      link points to is not removed.
 
+.. function:: directory-contents
+
+   Returns a list of files and subdirectories contained in a directory.
```

Comment:
Returns a _sequence_ of ...

Reply:
Thanks. I also added that `.` and `..` are not included in the result, for the avoidance of doubt.